### PR TITLE
KOGITO-6513 Add Github Issue Templates

### PR DESCRIPTION
**Pull request by r00ta**
Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.md)
- [x] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config)
- [] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

<details>
<summary>
How to replicate CI configuration locally?
</summary>

Build Chain tool does "simple" maven build(s), the builds are just Maven commands, but because the repositories relates and depends on each other and any change in API or class method could affect several of those repositories there is a need to use [build-chain tool](https://github.com/kiegroup/github-action-build-chain) to handle cross repository builds and be sure that we always use latest version of the code for each repository.
 
[build-chain tool](https://github.com/kiegroup/github-action-build-chain) is a build tool which can be used on command line locally or in Github Actions workflow(s), in case you need to change multiple repositories and send multiple dependent pull requests related with a change you can easily reproduce the same build by executing it on Github hosted environment or locally in your development environment. See [local execution](https://github.com/kiegroup/github-action-build-chain#local-execution) details to get more information about it.
</details>

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Run all builds</b>  
  Please add comment: <b>Jenkins retest this</b>

* <b>Run (or rerun) specific test(s)</b>  
  Please add comment: <b>Jenkins (re)run [kogito-runtimes|optaplanner|kogito-apps|kogito-examples|optaplanner-quickstarts|optaweb-employee-rostering|optaweb-vehicle-routing] tests</b>
 
* <b>Quarkus LTS checks</b>  
  Please add comment: <b>Jenkins run LTS</b>

* <b>Run (or rerun) LTS specific test(s)</b>  
  Please add comment: <b>Jenkins (re)run [kogito-runtimes|optaplanner|kogito-apps|kogito-examples|optaplanner-quickstarts|optaweb-employee-rostering|optaweb-vehicle-routing] LTS</b>

* <b>Native checks</b>  
  Please add comment: <b>Jenkins run native</b>

* <b>Run (or rerun) native specific test(s)</b>  
  Please add comment: <b>Jenkins (re)run [kogito-runtimes|optaplanner|kogito-apps|kogito-examples|optaplanner-quickstarts|optaweb-employee-rostering|optaweb-vehicle-routing] native</b>

* <b>Full Kogito testing</b> (with cloud images and operator BDD testing)  
  Please add comment: <b>Jenkins run BDD</b>  
  <b>This check should be used only if a big change is done as it takes time to run, need resources and one full BDD tests check can be done at a time ...</b>
</details>

### CODE REVIEW

**Comment by mareknovotny** [comment]
i had some request for a template from @ricardozanini here https://github.com/kiegroup/kogito-runtimes/pull/1485 it fell down from review table, so if you refresh it here i can close my PR.

**Review by mareknovotny — APPROVED** [review]
looks good

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+name: Bug Report
+description: Report a bug in Kogito
+labels: kind/bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! 
+        Please note that our official bug tracker is [here](https://issues.redhat.com/projects/KOGITO). But if for any reason you can not use it, feel free to fill out this form.
```
Comment (evacchi) [inline .github/ISSUE_TEMPLATE/bug_report.yml:9]:
```suggestion
        The preferred method to file a bug is [our JIRA board](https://issues.redhat.com/projects/KOGITO). You are still allowed to submit a GitHub issue, but you may experience delays in the feedback.
```

**Review by evacchi — APPROVED** [review]
small suggestion

**Comment by evacchi** [comment]
@r00ta I took the liberty to create the Jira issue for you

**Review by ricardozanini — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      label: Output of `uname -a` or `ver`
+  - type: input
+    attributes:
+      label:  Output of `java -version`
+  - type: input
+    attributes:
+      label:  GraalVM version (if different from Java)
+  - type: input
+    attributes:
+      label:  Kogito version or git rev
```
Comment (danielezonca) [inline .github/ISSUE_TEMPLATE/bug_report.yml:57]:
```suggestion
      label:  Kogito version or git rev (or at least Quarkus version if you are using Kogito via Quarkus platform BOM)
```

**Review by danielezonca — APPROVED** [review]
Just a minor comment but LGTM
Can you create a similar PR also for `kogito-apps` and `kogito-examples` too?

**Comment by r00ta** [comment]
@danielezonca 
https://github.com/kiegroup/kogito-operator/pull/1106
https://github.com/kiegroup/kogito-apps/pull/1194
https://github.com/kiegroup/kogito-examples/pull/1071
https://github.com/kiegroup/kogito-operator/pull/1106
https://github.com/kiegroup/kogito-tooling/pull/744

**Pull request merged**